editor/src/templates/image-drop.tsx

```diff
@@ -113,6 +115,14 @@ async function onDrop(
       'autoincrement',
     )
 
+    void saveAssets(context.editor.id!, assetInfo)
```

Comment:
should you await this before dispatching `saveImageActions: [EditorActions.updateFile...`?

Reply:
it could be awaited, my thinking was that

- the stuff you dragged to the canvas should appear ASAP, with minimal latency
- if something goes wrong, it's super easy to get back to the previous state with a ⌘-Z
no strong opinions though